users/admin.py

```diff
@@ -0,0 +1,51 @@
+"""
+Settings for admin site
+"""
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin as DefaultAdmin
+
+# from .forms import CustomUserChangeForm, CustomUserCreationForm
+# pylint: disable = no-member, protected-access
+from .models import User
+
+
+class UserAdmin(DefaultAdmin):
+    """
+    This class modifies the default options of User model for admin site
+    """
+
+    model = User
+    all_fields = [field.name for field in User._meta.fields]
+    all_fields.sort()
+    list_display = all_fields
```

Comment:
Let's display only `Id, Email, name, Is_staff, is_active, created_on` in the user's list in Django admin.

Reply:
Done